server/routes/label.js

```diff
@@ -0,0 +1,39 @@
+export default (app) => {
+  app
+    .get('/label', async (req, reply) => {
+      const labeles = await app.objection.models.label.query();
+      await reply.render('label/index', { currentUser: req.currentUser, labeles });
+    })
+    .get('/label/new', async (req, reply) => {
+      await reply.render('label/new', { data: {}, errors: [] });
+    })
+    .get('/label/edit/:id', async (req, reply) => {
+      const label = await app.objection.models.label.query().findById(req.params.id);
+      await reply.render('label/edit', { label, errors: [] });
+    })
+    .post('/label', async (req, reply) => {
+      try {
+        const newlabel = await app.objection.models.label.fromJson(req.body);
+        await app.objection.models.label.query().insert(newlabel);
+        await reply.redirect('/label');
+      } catch ({ data }) {
+        await reply.code(400).render('label/new', { data: req.body, errors: data });
```

Comment:
400 is:
> 400 Bad Request
    The server could not understand the request due to invalid syntax.

422 would fit better

Reply:
why is that? there are lines:
```
        const { id, name } = req.body;
        const updatedlabel = await app.objection.models.label.fromJson({ name });
```
so if client wont send name, that is badly formed request?

Reply:
https://tools.ietf.org/html/rfc7231#section-6.5.1
> The HyperText Transfer Protocol (HTTP) 400 Bad Request response status code indicates that the server cannot or will not process the request due to something that is perceived to be a client error (e.g., malformed request syntax, invalid request message framing, or deceptive request routing).

400 usually means that the request itself is maformed. Usually it's framework task to process such cases. 422 means that some provided data is invalid

Reply:
will do in https://github.com/alex-popov-tech/backend-project-lvl4/issues/19